lib/src/solid/api/rest_api.dart

```diff
@@ -355,202 +356,211 @@ Map<dynamic, dynamic> generateDefaultFiles(String appName) {
+//   final keyFileContent =
+//       await fetchPrvFile(keyFileUrl, accessToken, dPopTokenKeyFile);
+//   final keyFileDataMap = getFileContent(keyFileContent);
+
+//   // Define query parameters.
+
+//   final prefix1 = 'file: <$notesFile>';
+//   final prefix2 = 'notesTerms: <$notesTerms>';
+
+//   final subject = 'file:$resName';
+//   final predObjPath = 'notesTerms:$pathPred "$encNoteFilePath";';
+//   final predObjIv = 'notesTerms:$ivPred "$encNoteIv";';
+//   final predObjKey = 'notesTerms:$sessionKeyPred "$encSessionKey".';
+
+//   // Check if the resource is previously added or not.
+
+//   if (keyFileDataMap.containsKey(resName)) {
+//     final existPath = keyFileDataMap[resName][pathPred].toString();
+//     final existIv = keyFileDataMap[resName][ivPred].toString();
+//     final existKey = keyFileDataMap[resName][sessionKeyPred].toString();
+
+//     // If file does not contain the same encrypted value then delete and update
+//     // the file.
+//     // NOTE: Public key encryption generates different hashes different time for same plaintext value.
+//     // Therefore this always ends up deleting the previous and adding a new hash.
+//     if (existKey != encSessionKey ||
+//         existPath != encNoteFilePath ||
+//         existIv != encNoteIv) {
+//       final predObjPathPrev = 'notesTerms:$pathPred "$existPath";';
+//       final predObjIvPrev = 'notesTerms:$ivPred "$existIv";';
+//       final predObjKeyPrev = 'notesTerms:$sessionKeyPred "$existKey".';
+
+//       // Generate update sparql query.
+
+//       final query =
+//           'PREFIX $prefix1 PREFIX $prefix2 DELETE DATA {$subject $predObjPathPrev $predObjIvPrev $predObjKeyPrev}; INSERT DATA {$subject $predObjPath $predObjIv $predObjKey};';
+
+//       // Generate DPoP token.
+
+//       final dPopTokenKeyFilePatch =
+//           genDpopToken(keyFileUrl, rsaKeyPair, publicKeyJwk, 'PATCH');
+
+//       // Run the query.
+
+//       createUpdateRes = await updateFileByQuery(
+//           keyFileUrl, accessToken, dPopTokenKeyFilePatch, query);
+//     } else {
+//       // If the file contain same values, then no need to run anything.
+//       createUpdateRes = 'ok';
+//     }
+//   } else {
+//     // Generate insert only sparql query.
+
+//     final query =
+//         'PREFIX $prefix1 PREFIX $prefix2 INSERT DATA {$subject $predObjPath $predObjIv $predObjKey};';
+
+//     // Generate DPoP token.
+
+//     final dPopTokenKeyFilePatch =
+//         genDpopToken(keyFileUrl, rsaKeyPair, publicKeyJwk, 'PATCH');
+
+//     // Run the query.
+
+//     createUpdateRes = await updateFileByQuery(
+//         keyFileUrl, accessToken, dPopTokenKeyFilePatch, query);
+//   }
+
+//   if (createUpdateRes == 'ok') {
+//     return createUpdateRes;
+//   } else {
+//     throw Exception('Failed to create/update the shared file.');
+//   }
+// }
+
+// comment out the following function as it is not used in the current version
```

Comment:
Hi @anushkavidanage , could you please have a look at the following initialProfileUpdate function , if it is not needed, I can delete them. If we need to use them in the future, we can comment out just like what I did. :)